opencga-app/app/scripts/azure/arm/mongodb/azuredeploy.json

```diff
@@ -0,0 +1,258 @@
+                    "linuxConfiguration": {
+                        "disablePasswordAuthentication": false,
+                        "provisionVMAgent": true
+                    },
+                    "adminUsername": "[parameters('adminUsername')]",
+                    "adminPassword": "[parameters('adminPassword')]",
+                    "allowExtensionOperations": true
+                }
+            },
+            "tags": {}
+        },
+        {
+            "type": "Microsoft.Compute/virtualMachines/extensions",
+            "name": "[concat(variables('vmName'), '/configureagent')]",
+            "apiVersion": "2017-12-01",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[concat('Microsoft.Compute/virtualMachines/', variables('vmName'))]"
+            ],
+            "properties": {
+                "publisher": "Microsoft.Azure.Extensions",
+                "type": "CustomScript",
+                "typeHandlerVersion": "2.0",
+                "autoUpgradeMinorVersion": true,
+                "settings": {
+                    "timestamp": 4
+                },
+                "protectedSettings": {
+                    "commandToExecute": "[concat(' MONGODB_USERNAME=', variables('mongoDBUser'),' MONGODB_PASSWORD=', variables('mongoDBPassword'),' APP_DNS_NAME=', reference(resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPAddressName')),'2017-10-01').dnsSettings.fqdn, ' /bin/bash /opt/installmongodb.sh')]"
+                }
+            }
+        }
+    ],
+    "outputs": {
+        "mongDBUser": {
+            "value": "[variables('mongoDBUser')]",
+            "type": "string"
+        },
+        "mongDBPassword": {
+            "value": "[variables('mongoDBPassword')]",
+            "type": "string"
+        },
+        "fqdn": {
```

Comment:
be consistent with naming... other outputs have mongo prefix - should they? should this?

Reply:
Added mongo prefix in https://github.com/opencb/opencga/pull/975/commits/1ec41d9d76237c8182b43e1b401c073730cfff12